dist/README.md

```diff
@@ -0,0 +1,30 @@
+# Dist
+
+The `dist` folder contains sample configs for various platforms.
+
+## Conventions
+
+On unixes we will place agent configs under `/etc/nomad` and store data under `/var/lib/nomad/`. You will need to create both of these directories. We assume that `nomad` is installed to `/usr/local/bin/nomad`.
```

Comment:
I agree with @diptanu, it is a small detail but it is ideal with keep consistency with [FHS](http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html) to make admin's lives easier: